src/setup-guild.js

```diff
@@ -1,11 +1,12 @@
 const Discord = require('discord.js');
 const { REST } = require('@discordjs/rest');
 const { Routes } = require('discord-api-types/v10');
-const util = require('./util');
+const timedUtils = require('./utils/timed');
```

Comment:
The `errorCode` utils function is imported directly and not the entire module, but `timed` imports the entire module and then uses the function with dot notation. This is inconsistent. Either `errorCode` needs to be imported as a whole module and use dot notation like here, or `timed` needs to import only the function with destructuring